.github/workflows/ci.yml

```diff
@@ -74,9 +74,11 @@ jobs:
 #          ./run.sh
       - name: Check install script SHA-256 sum
         run: |
-          SHA256SUM_IN_SRC=$(grep "InstallScriptSHA = " env.go | \
-            grep -Eo "[0-9a-f]{64}")
-          SHA256SUM_INSTALLER=$(./bin/go run ./cmd/geninstaller \
-            --dest=$(mktemp) \
-            --dist-url=https://github.com/cashapp/hermit/releases/download/stable)
-          [ "$SHA256SUM_IN_SRC" = "$SHA256SUM_INSTALLER" ] || exit 1
+          for channel in "stable" "canary"; do
+            SHA256SUM_IN_SRC=$(grep "\"${channel}\":" env.go | \
+              grep -Eo "[0-9a-f]{64}")
+            SHA256SUM_INSTALLER=$(./bin/go run ./cmd/geninstaller \
+              --dest=$(mktemp) \
+              --dist-url=https://github.com/cashapp/hermit/releases/download/${channel})
+            [ "$SHA256SUM_IN_SRC" = "$SHA256SUM_INSTALLER" ] || exit 1
```

Comment:
Could `geninstaller` do this for check itself?

Reply:
The purpose of this sanity check is to ensure Hermit (and in turn `bin/hermit`) and `geninstaller` have coherent installer SHA-256 values. Because it's my understanding that `Hermit` and `geninstaller` are pretty much decoupled, letting `geninstaller` probe what `Hermit` is aware of does not seem to be straightforward (or elegant).